# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1316994075 14400
#      Sun Sep 25 19:41:15 2011 -0400
# Node ID 931d4b7dc205217f0fa86a5b5934500313aea536
# Parent  82f3a0c27569b87d68955e01151d445b13dea6db
avoid output in test

* uimenu.m: Avoid output in test.

diff --git a/scripts/plot/uimenu.m b/scripts/plot/uimenu.m
--- a/scripts/plot/uimenu.m
+++ b/scripts/plot/uimenu.m
@@ -135,14 +135,14 @@ endfunction
 %!   assert (ishghandle (uih))
 %! unwind_protect_cleanup
 %!   close (hf);
 %! end_unwind_protect
 
 %!test
 %! hf = figure ("visible", "off");
 %! unwind_protect
-%!   uie = findall (hf, "label", "&edit")
+%!   uie = findall (hf, "label", "&edit");
 %!   myui = uimenu (uie, "label", "mylabel");
 %!   assert (ancestor (myui, "uimenu", "toplevel"), uie)
 %! unwind_protect_cleanup
 %!   close (hf);
 %! end_unwind_protect
